### ISSUE TITLE
Purpose of 'VTScan'

### ISSUE THREAD

**Issue by imethanlee** [lindsey98/PhishIntention#11]
Hi,

I was wondering what is the purpose of calling VTscan when the model reports phishing ([code](https://github.com/lindsey98/PhishIntention/blob/e1e2d90de8196b60df8df5fe839ae4ea2bf78d66/phishintention/phishintention_main.py#L187)). Since it has no influence on the phishing detection result, I guess it just serves as an additional information?

Thanks.

**Comment by lindsey98** [comment]
Yes, it has no influence. We just want to check how many zero-day phishing we have reported, i.e., not reported by VirusTotal.

**Comment by imethanlee** [comment]
Thanks!
